frontend/src/components/calendarView/Calendar.jsx

```diff
@@ -33,41 +32,85 @@ const CalendarTable = ({ data, startDate, addDataFunc }) => {
       <table className="calendar">
         <thead>
           <tr>
-            <td/>
+            <td />
             {dateList.map((date) => (
-              <th key={date}>{new Intl.DateTimeFormat('en-US', options).format(new Date(date))}</th>
+              <th key={date}>
+                {new Intl.DateTimeFormat("en-US", options).format(
+                  new Date(date)
+                )}
+              </th>
             ))}
           </tr>
         </thead>
         <tbody>
           <tr>
-          <td>Fajr</td>
+            <td>Fajr</td>
             {dateList.map((date) => (
-              <td className={`prayer-count-${data[date] ? Object.keys(data[date]).length : 0}`} key={`${date}-fajr`} onClick={() => addDataFunc(date, "fajr")}>{data[date]?.["fajr"]}</td>
+              <td
+                className={`prayer-count-${
+                  data[date] ? Object.keys(data[date]).length : 0
+                }`}
+                key={`${date}-fajr`}
+                onClick={() => addDataFunc(date, "Fajr")}
+              >
+                {data[date]?.["Fajr"]}
+              </td>
```

Comment:
Should we change this to one component where you give it attributes to differentiate each prayer